portafly/src/i18n/locales/en/audience/account/account.yml

```diff
@@ -0,0 +1,58 @@
+---
+# Accounts > Account overview
+
+page_title: Account
+page_title_description: Page title of the subsection for Accounts Overview
+
+# Top area
+
+body_title: "Account: {{organization_name}}"
```

Comment:
@porueesq @dfennessy 
To be consistent with the other pages (eg: Product ). We have decided to use` {{organization_name}} ` as the page title. Since the user could find where he is through the breadcrumbs later.
So we can remove "Account: "